build/configs/rtl8730e/Make.defs

```diff
@@ -54,7 +54,12 @@ include ${TOPDIR}/tools/Config.mk
 ARCH_FAMILY = $(patsubst "%",%,$(CONFIG_ARCH_FAMILY))
 include ${TOPDIR}/arch/$(CONFIG_ARCH)/src/$(ARCH_FAMILY)/Toolchain.defs
 
+ifeq ($(CONFIG_RTL8730E_BOARD_REVISION),6)

```

Comment:
This condition is for revision 6. but config is `CONFIG_RTL8730E_BOARD_REVISION=7` in defconfig.

Reply:
I think it should be changed to `ifeq ($(CONFIG_RTL8730E_BOARD_REVISION), $(shell expr $(CONFIG_RTL8730E_BOARD_REVISION) \> 7))` to use ddr ld file for ddr board (revision 7)

Reply:
I couldn't do numerical compare in .defs file.
Will temporary use ifeq ($(CONFIG_RTL8730E_BOARD_REVISION),7)